vhdbuilder/packer/vhd-scanning.sh

```diff
@@ -4,12 +4,16 @@ set -eux
 TRIVY_SCRIPT_PATH="trivy-scan.sh"
 EXE_SCRIPT_PATH="vhd-scanning-exe-on-vm.sh"
 TEST_RESOURCE_PREFIX="vhd-scanning"
-VM_NAME="$TEST_RESOURCE_PREFIX-vm"
+VM_NAME="$TEST_RESOURCE_PREFIX-vm-$(date +%s)-$RANDOM"
 VHD_IMAGE="$MANAGED_SIG_ID"
 
 SIG_CONTAINER_NAME="vhd-scans"
 TEST_VM_ADMIN_USERNAME="azureuser"
 
+# we must create VMs in a vnet which has access to the storage account, otherwise they will not be able to access the VHD blobs
+VNET_NAME="nodesig-pool-vnet-${PACKER_BUILD_LOCATION}"
+SUBNET_NAME="scanning"
```

Comment:
Just curious, is the subnet unique for each usecase where we reach private endpoint?

Reply:
@jason1028kr we will have two subnets in the vnet for every location where we perform the build. They can be reused across builds and are created when the build pool is provisioned (so outside of the scope of this script)